Update/overhaul (#3)
flupes/svplume: projects/project-anchors.md

---
title: Anchoring system
permalink: /projects/project-anchors/
layout: default
---

## Anchoring System

### Main Anchor

Since 2002, our primary anchor has been a Spade A80 aluminum model. It weighs
just 15 lbs but has a surface large enough to hold boats up to 34 ft. It quickly
became our favorite: light enough that even Cécile can raise it, yet extremely
reliable. Over the years it never once let us down. 

However, in 2020, we noticed some oxidation forming between the ballast and the
aluminum. When we reached out to Spade about it, they responded by offering us a
40% discount on a brand-new anchor (see picture below).

Our main anchor is set up with 120 ft of 1/4” ACCO Grade 43 chain and 150 ft of
1/2” Samson Super Strong double-braid nylon line, giving us a total of 270 ft of
rode.

{% picture projects/project-anchor-main-spade.jpg --alt Spade A80 Anchor --img class="img-small"%}
<p style="text-align: center; font-style: italic;">
  Spade A80 main anchor (15 lbs)
</p>

### Storm Anchor

> In addition, we carry a storm anchor just in case, a M2 Mantus of 35 lbs. It is recommended for boats up to 30 ft in winds greater than 50 knt! It is easy to store as it breaks down into small parts. A single bolt is used for disassembly and storage. Ours is securely fixed to the cabin sole, yet easily removable, as shown in the pictures below.

Reviewer: ```suggestion
In addition, we carry a storm anchor just in case, a M2 Mantus of 35 lbs. It is recommended for boats up to 30 ft in winds greater than 50 knt! It is easy to store as the shank can be disassembled for storage. Ours is securely fixed to the cabin sole, yet easily removable, as shown in the pictures below.
```
